test/e2e/cli/helm/helm.go

```diff
@@ -107,6 +107,8 @@ func InstallOperatorWideSubmodule(input model.UserInputs) {
 		"atlas-operator-"+input.Project.GetProjectName(),
 		config.AtlasOperatorHelmChartPath,
 		"--set-string", fmt.Sprintf("atlasURI=%s", config.AtlasHost),
+		"--set", "objectDeletionProtection=false",
+		"--set", "subobjectDeletionProtection=false",
```

Comment:
I beleve all this changes need to go away after the rebase, those flags do not exist anymore.